[01042022 17:44:09] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[01042022 17:44:09] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[01042022 17:45:10] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 17:46:10] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 17:47:10] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 17:48:10] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 17:49:05] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:83-warning: DATABASE opening information_schema
[01042022 17:49:05] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:86-warning: open db information_schema -> :memory:
[01042022 17:49:10] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 17:49:36] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[01042022 17:50:10] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 17:50:47] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:83-warning: DATABASE opening information_schema
[01042022 17:50:47] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:86-warning: open db information_schema -> :memory:
[01042022 17:51:10] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 17:52:10] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 17:52:45] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
[01042022 17:52:48] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[01042022 17:52:48] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[01042022 17:53:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 17:54:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 17:55:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 17:56:41] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
[01042022 17:56:44] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[01042022 17:56:44] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[01042022 17:57:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 17:58:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 17:59:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:00:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:01:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:02:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:03:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:04:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:05:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:06:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:07:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:08:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:09:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:10:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:11:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:12:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:13:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:14:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:15:38] ftp /opt/dionaea/lib/dionaea/python/dionaea/ftp.py:235-warning: cmd 'b'\x16\x03\x01\x00\xb0\x01\x00\x00\xac\x03\x03\x80\xa3''
[01042022 18:15:38] ftp /opt/dionaea/lib/dionaea/python/dionaea/ftp.py:235-warning: cmd 'b'\xd2\xa4\xf2\x12\x9b\xceu\x8b+\xbf\xe7\x99\xc3]\xc8q\xeb\x12\xfbP\x18/\x1do\x06\xf3R\xe8\x81\x00\x00\x1a\xc0/\xc0+\xc0\x11\xc0\x07\xc0\x13\xc0\t\xc0\x14\xc0''
[01042022 18:15:38] ftp /opt/dionaea/lib/dionaea/python/dionaea/ftp.py:235-warning: cmd 'b'\x00\x05\x00/\x005\xc0\x12\x00''
[01042022 18:15:38] ftp /opt/dionaea/lib/dionaea/python/dionaea/ftp.py:235-warning: cmd 'b'\x01\x00\x00i3t\x00\x00\x00\x00\x00\x12\x00\x10\x00\x00''
[01042022 18:15:38] ftp /opt/dionaea/lib/dionaea/python/dionaea/ftp.py:235-warning: cmd 'b'34.121.180.85\x00\x05\x00\x05\x01\x00\x00\x00\x00\x00''
[01042022 18:15:38] ftp /opt/dionaea/lib/dionaea/python/dionaea/ftp.py:235-warning: cmd 'b'\x00\x08\x00\x06\x00\x17\x00\x18\x00\x19\x00\x0b\x00\x02\x01\x00\x00''
[01042022 18:15:38] ftp /opt/dionaea/lib/dionaea/python/dionaea/ftp.py:235-warning: cmd 'b'\x00\x10\x00\x0e\x04\x01\x04\x03\x02\x01\x02\x03\x04\x01\x05\x01\x06\x01\xff\x01\x00\x01\x00\x00\x10\x00\x17\x00\x15\x02h2\x08http/1.1\x08spdy/3.1\x15\x03\x01\x00\x02\x02''
[01042022 18:15:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:15:49] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[01042022 18:16:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:17:39] ftp /opt/dionaea/lib/dionaea/python/dionaea/ftp.py:235-warning: cmd 'b'HELP''
[01042022 18:17:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:18:18] ftp /opt/dionaea/lib/dionaea/python/dionaea/ftp.py:235-warning: cmd 'b'HELP''
[01042022 18:18:20] ftp /opt/dionaea/lib/dionaea/python/dionaea/ftp.py:235-warning: cmd 'b'FEAT''
[01042022 18:18:23] ftp /opt/dionaea/lib/dionaea/python/dionaea/ftp.py:235-warning: cmd 'b'USER''
[01042022 18:18:26] ftp /opt/dionaea/lib/dionaea/python/dionaea/ftp.py:235-warning: cmd 'b'PASS''
[01042022 18:18:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:19:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:20:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:21:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:22:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:23:27] ftp /opt/dionaea/lib/dionaea/python/dionaea/ftp.py:235-warning: cmd 'b'GET''
[01042022 18:23:27] ftp /opt/dionaea/lib/dionaea/python/dionaea/ftp.py:235-warning: cmd 'b'Host:''
[01042022 18:23:27] ftp /opt/dionaea/lib/dionaea/python/dionaea/ftp.py:235-warning: cmd 'b'User-Agent:''
[01042022 18:23:27] ftp /opt/dionaea/lib/dionaea/python/dionaea/ftp.py:235-warning: cmd 'b'Accept-Encoding:''
[01042022 18:23:27] ftp /opt/dionaea/lib/dionaea/python/dionaea/ftp.py:235-warning: cmd 'b'Accept:''
[01042022 18:23:27] ftp /opt/dionaea/lib/dionaea/python/dionaea/ftp.py:235-warning: cmd 'b'Connection:''
[01042022 18:23:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:24:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:25:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:26:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:27:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:28:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:29:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:30:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:31:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:32:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:33:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:34:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:35:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:36:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:37:11] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
[01042022 18:37:15] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[01042022 18:37:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[01042022 18:38:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:39:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:40:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:41:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:42:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:43:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:44:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:45:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:46:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:47:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:48:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:49:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:50:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:51:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:52:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:53:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:54:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:55:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:55:17] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
[01042022 18:55:21] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[01042022 18:55:21] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[01042022 18:56:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:57:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:58:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 18:59:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 19:00:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 19:01:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 19:02:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 19:03:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 19:04:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 19:05:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 19:06:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 19:07:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 19:08:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 19:09:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 19:10:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 19:11:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 19:12:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 19:13:17] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[01042022 19:13:17] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[01042022 19:13:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 19:14:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 19:14:54] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
[01042022 19:15:05] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[01042022 19:15:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[01042022 19:16:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 19:17:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 19:18:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 19:19:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 19:20:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 19:21:01] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[01042022 19:21:01] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[01042022 19:21:02] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[01042022 19:21:03] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[01042022 19:21:03] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[01042022 19:21:04] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[01042022 19:21:05] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[01042022 19:21:06] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[01042022 19:21:07] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 19:21:07] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[01042022 19:21:08] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[01042022 19:21:09] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[01042022 19:21:10] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[01042022 19:21:11] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[01042022 19:22:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 19:23:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 19:24:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 19:25:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 19:26:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 19:27:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 19:28:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 19:29:04] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
[01042022 19:29:09] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[01042022 19:29:09] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[01042022 19:30:10] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 19:31:10] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 19:32:10] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 19:33:10] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 19:33:24] ftp /opt/dionaea/lib/dionaea/python/dionaea/ftp.py:235-warning: cmd 'b'AUTH''
[01042022 19:33:24] ftp /opt/dionaea/lib/dionaea/python/dionaea/ftp.py:235-warning: cmd 'b'AUTH''
[01042022 19:34:10] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 19:35:10] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 19:36:10] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 19:37:10] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 19:39:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 19:42:08] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[01042022 19:42:08] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 19:42:08] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[01042022 19:42:08] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 19:42:08] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 19:42:08] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[01042022 20:58:58] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
